python/tests/test_options.py

```diff
@@ -12,13 +12,20 @@ def test_defaults_to_win10_chrome_latest(self):
         assert sauce.browserVersion == 'latest'
         assert sauce.platformName == 'Windows 10'
 
-    def test_accepts_browser_version_platform_name(self):
+    def test_accepts_browser_name(self):
         sauce = SauceOptions('Firefox')
 
         assert sauce.browserName == 'Firefox'
         assert sauce.browserVersion == 'latest'
         assert sauce.platformName == 'Windows 10'
 
+    def test_accepts_browser_version_platform_name(self):
+        sauce = SauceOptions(browserName='Chrome', browserVersion='75.0', platformName='macOS 10.13')
+
+        assert sauce.browserName == 'chrome'
+        assert sauce.browserVersion == '75.0'
+        assert sauce.platformName == 'macOS 10.13'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626732)